packages/reports/addon/templates/components/grouped-list.hbs

```diff
@@ -1,7 +1,9 @@
-{{!-- Copyright 2017, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+{{!-- Copyright 2019, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
 {{#each-in groupedItems as | group items |}}
   <details class="grouped-list__group" open={{shouldOpenAllGroups}}>
-    <summary class="grouped-list__group-header">{{group}} ({{items.length}})</summary>
+    <summary class="fa grouped-list__group-header">
+      <span class="grouped-list__group-header-content">{{group}} ({{items.length}})</span>
```

Comment:
Do we need the `span` here?

Reply:
I've added the `fa` class to the parent `summary` for the open/close arrows in the `:before`
The inner span is mainly to override the fa font
The other option would be not to add the `fa` and copy all the `fa` styles to the `:before`